flags/templatetags/feature_flags.py

```diff
@@ -0,0 +1,31 @@
+from django import template
+
+from flags.template_functions import (
+    flag_enabled as base_flag_enabled,
+    flags_enabled as base_flags_enabled,
+    flag_disabled as base_flag_disabled
+)
+
+
+register = template.Library()
+
+
+# @register.simple_tag(takes_context=True)
```

Comment:
Right now we're supporting Django 1.8 and up. Once we get to 1.9 [we can just `simple_tag` for assignment](https://docs.djangoproject.com/en/1.10/howto/custom-template-tags/#assignment-tags).